src/lib/parseOffenceReason.test.ts

```diff
@@ -0,0 +1,90 @@
+import parseOffenceReason from "./parseOffenceReason"
+import { COMMON_LAWS, INDICTMENT } from "./properties"
+
+describe("GIVEN parseOffence", () => {
+  describe("WHEN parsing a national offenceCode", () => {
+    it("THEN parse an offenceCode with a COMMON_LAWS prefix", () => {
+      const commonLawOffenceCode = `${COMMON_LAWS}001A`
+      const res = parseOffenceReason(commonLawOffenceCode, "01")
+      expect(res).toStrictEqual({
+        __type: "NationalOffenceReason",
+        OffenceCode: {
+          __type: "CommonLawOffenceCode",
+          CommonLawOffence: COMMON_LAWS,
+          Reason: "ML001",
+          Qualifier: "A",
+          FullCode: `${COMMON_LAWS}001A`
+        }
+      })
+    })
+
+    it.skip("THEN parse an offenceCode with a INDICTMENT prefix", () => {
```

Comment:
Skipped because there are no Indictment codes in the `offence-code.json` so the test will fail as it can't be confirmed against the json.